projects/smart-table/src/lib/smart-table.module.ts

```diff
@@ -17,7 +17,8 @@ const defaultConfiguration: IModuleConfig = {
   storageType: 'localStorage',
   identifier: 'aui-smart-table'
 };
-
+// Do not remove following line as this line prevents build errors for the smart table module
```

Comment:
Can you maybe give some background on this via a link?

Reply:
https://github.com/angular/angular/issues/26874